service/src/main/java/bio/terra/workspace/service/policy/TpsRetry.java

```diff
@@ -20,9 +20,9 @@ public class TpsRetry {
 
   // The retry function starts with INITIAL_WAIT between retries, and doubles that until it
   // reaches MAXIMUM_WAIT, after which all retries are MAXIMUM_WAIT apart.
-  private static final Duration MAXIMUM_WAIT = Duration.ofSeconds(30);
-  private static final Duration INITIAL_WAIT = Duration.ofSeconds(10);
-  private static final Duration OPERATION_TIMEOUT = Duration.ofSeconds(300);
+  private static final Duration MAXIMUM_WAIT = Duration.ofSeconds(1);
```

Comment:
I temporarily bumped these retries down to 1 second, which was helpful with iteration speed, but plan to revert this.  Leaving this comment here as a reminder to my future self that this was helpful.